core/buffer.py

```diff
@@ -270,3 +270,6 @@ def fake_key_event(self, event_string):
 
     def update_settings(self):
         pass
+
+    def get_bookmark(self):
+        pass
```

Comment:
This need change to

```Python
def get_bookmark(self):
    return ""
```

Because call_function DBus always need get_bookmark return string.